Tests/RockLib.Configuration.Remote.Tests/RemoteConfigurationSourceTests.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.Extensions.Configuration;
+using Moq;
+using Xunit;
+
+namespace RockLib.Configuration.Remote.Tests;
+
+#pragma warning disable CA2000 // Dispose objects before losing scope
```

Comment:
I don't remember if it would have been a significant lift, or if I just didn't want to address that _yet_. It feels of relatively low importance for a unit test,  but I'll take another look and see if it can be addressed properly.